File: minio/test/python/put.py
#!/bin/env python
from minio import Minio
from minio.error import ResponseError

import os 
import tempfile


IPADDR = "10.102.1.140:9000"
ACCESS_KEY = "minio"
SECRETKEY = "miniostorage"


minioClient = Minio(IPADDR,
              access_key=ACCESS_KEY,
              secret_key=SECRETKEY,
              secure=False)
try:
    print(minioClient.fput_object('mybucket', 'myobject', '/tmp/test'))
except ResponseError as err:
    print(err)

File: minio/test/go/put.go
package main

import (
	"github.com/minio/minio-go"
	"log"
    "fmt"
)

func main() {
	endpoint := "10.102.1.140:9000"
	accessKeyID := "minio"
	secretAccessKey := "miniostorage"
	useSSL := false

	// Initialize minio client object.
	minioClient, err := minio.New(endpoint, accessKeyID, secretAccessKey, useSSL)
	if err != nil {
		log.Fatalln(err)
	}
	log.Printf("%#v\n", minioClient)
	n, err := minioClient.FPutObject("mybucket", "myobject.csv", "/tmp/test", "application/text")
	if err != nil {
		fmt.Println(err)
		return
	}
    fmt.Println(n)
}
